src/layer-composer/generators/basemap/basemap-layers.ts

```diff
@@ -1,27 +1,73 @@
-import { Group } from '../../../types'
+import { Group, Dictionary } from '../../../types'
+import { Layer, AnySourceImpl } from 'mapbox-gl'
 
-const layers = {
-  'north-star': {
-    source: {
-      tiles: [
-        'https://mt1.google.com/vt/lyrs=s&x={x}&y={y}&z={z}',
-        'https://mt2.google.com/vt/lyrs=s&x={x}&y={y}&z={z}',
-        'https://mt3.google.com/vt/lyrs=s&x={x}&y={y}&z={z}',
-      ],
-      type: 'raster',
-      tileSize: 256,
+export const BASEMAPS = {
+  Satellite: 'satellite',
+  Landmass: 'landmass',
+  Graticules: 'graticules',
+}
+
+const BASEMAP_VECTOR_SOURCE = 'basemap_vector'
+
+export const layers: Dictionary<Layer> = {
+  [BASEMAPS.Satellite]: {
+    type: 'raster',
+    id: BASEMAPS.Satellite,
+    source: BASEMAPS.Satellite,
+    metadata: {
+      group: Group.Basemap,
+    },
+  },
+  [BASEMAPS.Landmass]: {
+    type: 'fill',
+    id: BASEMAPS.Landmass,
+    source: BASEMAP_VECTOR_SOURCE,
+    'source-layer': 'landmass',
```

Comment:
Why are you duplicating the string here?
```suggestion
    'source-layer':  BASEMAPS.Landmass,
```